x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/shared/SharedBlobCacheService.java

```diff
@@ -623,6 +629,7 @@ public void forceEvict(Predicate<KeyType> cacheKeyPredicate) {
                 }
             }
         }
+        return matchingEntries.size();
```

Comment:
When performing an side-effecting (mutation of state) operation, it's nice to provide _feedback_ when something is done. This comes from the idea of feedback in UX design. Even if we don't use this value, it can be used to show a  (human) callee that an effect took place.

Reply:
We should be counting only when `evicted` is true

Reply:
Right, good catch 👍